### DISCUSSION TITLE
News duplicated shortcodes

### DISCUSSION THREAD

**Discussion by rica-carv** [e107inc/e107#5293] [Q&A Plugins]
There are a few news shortcodes with the same output:
- `{NEWS_CATEGORY_TITLE}` is equal to `{NEWS_CATEGORY_NAME}`
- `{NEWS_CATEGORY_URL}` is equal to `{NEWS_NAV_URL: items=category}`

Is this intended?

**Comment by Vodhin** [comment]
Probably for backwards compatibility with older themes/plugins/mods.
